metrics/thresholds_test.go

```diff
@@ -638,12 +638,21 @@ func TestThresholdsRunAll(t *testing.T) {
 	}
 }
 
-func TestThresholds_Run(t *testing.T) {
+func TestThresholdsRun(t *testing.T) {
 	t.Parallel()
 
+	// The median is calculated by the `TrendSink.Calc` method.
+	// If values are not added to the sink via its `Add` method,
+	// the sinks `jumbled` flag will remain false.
+	filledTrendSink := &TrendSink{}
+	filledTrendSink.Add(Sample{Time: time.Now(), Value: 80})
+	filledTrendSink.Add(Sample{Time: time.Now(), Value: 80})
+	filledTrendSink.Add(Sample{Time: time.Now(), Value: 80})
```

Comment:
Having all values the same looks like one of the possible test cases, but I believe it's better to have a case where they are diverse. WDYT?

Reply:
Fair enough 👍🏻 I'll switch to `80`, `90`, `100` 😄

Reply:
It seems like this was affected by rebase to :cry:

Reply:
Arf 😞 I need to get better at this 🙇🏻